ansible_collections/arista/avd/docs/installation/requirements.md

```diff
@@ -17,7 +17,7 @@ If you leverage [CloudVision](https://www.arista.com/en/products/eos/eos-cloudvi
 
 - ansible-core from __2.11.3__ to __2.12.x__ excluding __2.12.0__ to __2.12.5__
 
-Excluded versions have an [issue](## "plugin loader will now load config data for plugin by name instead of by file to avoid issues with the same file being loaded under different names (fqcn + short name).")
+Excluded versions have an [issue](## "plugin loader will now load config data for plugin by name instead of by file to avoid issues with the same file being loaded under different names (fqdn + short name).")
```

Comment:
This was inded FQCN which is an Ansible term for fully-qualified-collection-name.
```suggestion
Excluded versions have an [issue](## "plugin loader will now load config data for plugin by name instead of by file to avoid issues with the same file being loaded under different names ([FQCN](https://docs.ansible.com/ansible/latest/reference_appendices/glossary.html#term-Fully-Qualified-Collection-Name-FQCN) + short name).")
```